building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+# Building a Preview
+
+- [Build a Preview](#build-a-preview)
+- [Share your Preview](#share-your-preview)
+- [Preview Actions](#preview-actions)
+- [Preview status](#preview-status)
+- [Set a Base Preview](#set-a-base-preview)
+- [Administer Base Previews](#administer-base-previews)
+- [Auto-generate Previews](#auto-generate-previews)
+- [Auto-delete Previews](#auto-delete-previews)
+- [Optimize your Preview builds](#optimize-your-preview-builds)
+- [Preview size explained](#preview-size-explained)
+
+## Build a Preview
+
+Once you've
+[set up your Tugboat project](../setting-up-tugboat/index.md#create-a-project),
+[linked a repo](../setting-up-tugboat/index.md#add-repos-to-the-project), and
+[put your Config file in the linked repo](../setting-up-tugboat/index.md#create-a-tugboat-config-file),
+it's time to build a Preview!
+
+- [How to build a Preview](#how-to-build-a-preview)
+- [The build process: explained](#the-build-process-explained)
+
+### How to build a Preview
+
+To build a Preview:
+
+1. Click your username in the upper right-hand corner, and go to Projects;
+2. Click the **My Projects** link;
+3. Select the project where you want to build a Preview;
+4. Click the name of the repo that contains the code you want to use to build
+   the Preview;
+5. Scroll down to the **Available to Build** section; by default, you'll see
+   _Branches_, but you can switch to view Tags or Pull Requests that are
+   available to Preview;
```

Comment:
> by default, you'll see
    _Branches_, but you can switch to view Tags or Pull Requests that are
    available to Preview

By default, you'll see 'pull requests', unless there are none, in which case you will see 'branches'